Updated the Readme file for the rcmanager (#333)
robocomp/robocomp: tools/rcmanagerSimple/README.md

[![Join the chat at https://gitter.im/robocomp/robocomp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/robocomp/robocomp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

## What the rcmanager does?

> The `rcmanager` is a very useful tool for controlling the components that we have running within ROBOCOMP. The `rcmanager` is basically a user interface that shows you a graph that relates the components of the RoboComp with nodes (of the graph) and relationships/dependencies between them (arrows of the graph).

Reviewer: ```suggestion
The `rcmanager` is a very useful tool for controlling the components that we have running within ROBOCOMP. The `rcmanager` is basically a user interface that shows you a graph that relates the RoboComp components with nodes (of the graph) and relationships/dependencies between them (arrows of the graph).
```

1. NODES: nodes (components) have two possible states:
    > DOWN: when the nodes are red. This means that the component is not being executed. If we want to run the component, we must click on the node with the right mouse button and select the UP option.
    > UP: when the nodes are green. This means that the component is being executed. If we want to stop the component, we have to click on the node with the right mouse button and select the DOWN option.

> 2. ARROWS: they are the dependencies between nodes. The sense of dependability is equal to the direction of the arrow. For example, in the first image we can see that the commonjoint component (which is responsible for handling all robot motors) depends on two other components: the dynamixel component (which is responsible for handling the head motors) and the faulhaber component (which is responsible for handling the arm motors).

Reviewer: ```suggestion
2. ARROWS: they are the dependencies between nodes. The direction of the arrow represents what depends on what. For example, in the first image, we can see that the commonjoint component (which is responsible for handling all robot motors) depends on two other components: the dynamixel component (which is responsible for handling the head motors) and the Faulhaber component (which is responsible for handling the arm motors).
```
